theano/tests/test_flake8.py

```diff
@@ -14,7 +14,7 @@
     flake8_available = False
 
 __authors__ = ("Saizheng Zhang")
-__copyright__ = "(c) 2015, Universite de Montreal"
+__copyright__ = "(c) 2016, Universite de Montreal"
```

Comment:
About this one, we wanted to do as in this file everywhere in Theano. But
we didn't do it. But as this is our license, I would update it. I'm not
sure if we should keep it, so by not being sure, I would just update it.

On Thu, Jan 7, 2016 at 2:32 PM, abergeron <notifications@github.com> wrote:

> In theano/tests/test_flake8.py
> <https://github.com/Theano/Theano/pull/3830#discussion_r49115026>:
>
> > @@ -14,7 +14,7 @@
> >      flake8_available = False
> >
> >  __authors__ = ("Saizheng Zhang")
> > -__copyright__ = "(c) 2015, Universite de Montreal"
> > +__copyright__ = "(c) 2016, Universite de Montreal"
>
> I'm not sure about this one ...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/3830/files#r49115026>.
>